jobs/gtfs-rt-parser-v2/gtfs_rt_parser.py

```diff
@@ -408,7 +409,10 @@ def validate_and_upload(
             [
                 {
                     # back and forth so we can use pydantic serialization
-                    "metadata": make_pydantic_model_bq_safe(extract),
+                    "metadata": {
+                        "gtfs_validator_version": GTFS_RT_VALIDATOR_VERSION,
+                        "extract": make_pydantic_model_bq_safe(extract),
```

Comment:
do we want to use the whole extract here? or also just filepath like we're doing in the other places?